src/VolleyManagement.WebClient/spa/app/Components/Tournaments/tournaments.component.html

```diff
@@ -0,0 +1,22 @@
+<table class="table">
+  <thead>
+    <tr>
+      <td width="400">
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this deprecated "width" attribute. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=Web%3AS1827)